Makefile

```diff
@@ -1,19 +1,23 @@
-ddown = docker-compose down
-
-install: build migrate
+# General =====================================================================
+install: build migrate import_initial_data
```

Comment:
Are you sure?

Reply:
I guess it's easier as it does everything if you just wanted to get up and running in one command